source/features/rgh-linkify-yolo-issues.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import {wrap} from '../helpers/dom-utils';
+import features from '../feature-manager';
+import {getRghIssueUrl} from '../helpers/rgh-issue-link';
+import {isRefinedGitHubYoloRepo} from '../github-helpers';
+
+function init(): void {
+	for (const issueCell of select.all('.js-csv-data td:nth-child(3)')) {
+		wrap(issueCell.firstChild!, <a href={getRghIssueUrl(issueCell.textContent!)}/>);
+	}
+}
+
+void features.add(import.meta.url, {
+	asLongAs: [
+		isRefinedGitHubYoloRepo,
+		pageDetect.isSingleFile,
+		() => location.pathname.endsWith('broken-features.csv'),
+	],
+	awaitDomReady: false,
```

Comment:
With this, select.all won't work. It needs to be dropped

Reply:
I have dropped the `awaitDomReady: false`. And tested it once, let me know if the selector is still causing an issue I will change it then.